README.md

```diff
@@ -1,7 +1,10 @@
-[![Build Status](https://travis-ci.org/HyperBrain/serverless-aws-alias.svg?branch=master)](https://travis-ci.org/HyperBrain/serverless-aws-alias)
-[![Coverage Status](https://coveralls.io/repos/github/HyperBrain/serverless-aws-alias/badge.svg?branch=master)](https://coveralls.io/github/HyperBrain/serverless-aws-alias?branch=master)
-[![npm version](https://badge.fury.io/js/serverless-aws-alias.svg)](https://badge.fury.io/js/serverless-aws-alias)
-[![npm](https://img.shields.io/npm/dt/serverless-aws-alias.svg)](https://www.npmjs.com/package/serverless-aws-alias)
+[![Serverless][ico-serverless]][link-serverless]
```

Comment:
Travis should work at the new location too, so we just can update the link and replace `hyperbrain` with `serverless-heaven`

Reply:
Oh, just saw that I missed it ;-) Ready to go from my side